spec/pytest_spec.vim

```diff
@@ -53,6 +53,16 @@ describe "PyTest_xunit"
     TestNearest
 
     Expect g:test#last_command == 'python -m pytest test_method.py::test_numbers'
+
+    view +5 test_class.py 
+    TestClass
+
+    Expect g:test#last_command == 'python -m pytest test_class.py::TestSubclass'
+
+    view +6 test_class.py 
+    TestClass
+
+    Expect g:test#last_command == 'python -m pytest test_class.py::TestSubclass'
```

Comment:
Could these TestClass examples please be removed out of the TestNearest block

Reply:
Moved.